pkgs/by-name/gr/gruvbox-plus-icons/package.nix

```diff
@@ -21,7 +21,7 @@ stdenvNoCC.mkDerivation {
 
   nativeBuildInputs = [ gtk3 ];
 
-  propagatedBuildInputs = [ breeze-icons gnome-icon-theme hicolor-icon-theme ];
+  propagatedBuildInputs = [ plasma5Packages.breeze-icons gnome-icon-theme hicolor-icon-theme ];
```

Comment:
Are these changes necessary? They seem to bringing more than just the icons into scope for an icon package.

Reply:
Yeah this is necessary because on master, this fails:
```
$ nix-instantiate -A gruvbox-plus-icons --arg config '{ allowAliases = false; }'
error:
       … while calling the 'throw' builtin

         at /home/tweagysil/src/nixpkgs/pr/lib/customisation.nix:206:13:

          205|        then makeOverridable f allArgs
          206|        else throw "lib.customisation.callPackageWith: ${error}";
             |             ^
          207|

       error: lib.customisation.callPackageWith: Function called without required argument "breeze-icons" at /home/tweagysil/src/nixpkgs/pr/pkgs/by-name/gr/gruvbox-plus-icons/package.nix:6
```

This is because `breeze-icons` is an alias. We can look into `aliases.nix` to see where it comes from: https://github.com/NixOS/nixpkgs/blob/989df07dc54ec214b324a720b5bdf056c3132a3e/pkgs/top-level/aliases.nix#L1087-L1089

So the change here just inlines the alias, therefore not using the alias anymore

Reply:
Ah I see